File: Data Structures & Algorithms/remove-element/submission-2.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        vector<int> copy;
        for (int num : nums) {
            if (num != val) {
                copy.push_back(num);
            }
        }
        nums = copy;

        return nums.size();
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-2.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        results = {}
        for i in range(len(nums)):
            num = nums[i]
            diff = target - num
            if diff in results:
                return [results[diff], i]
            results[num] = i